composer.json

```diff
@@ -13,30 +13,30 @@
     ],
     "require": {
         "php": "^7.1",
-        "symfony/framework-bundle": "^3.4|^4.3",
-        "symfony/console": "^3.4|^4.3",
-        "symfony/dependency-injection": "^3.4|^4.3",
-        "symfony/property-access": "^3.4|^4.3",
+        "symfony/framework-bundle": "^3.4|^4.3|^5",
+        "symfony/console": "^3.4|^4.3|^5",
+        "symfony/dependency-injection": "^3.4|^4.3|^5",
+        "symfony/property-access": "^3.4|^4.3|^5",
         "pagerfanta/pagerfanta": "^1.0.5|^2.0",
         "psr/log": "^1.0",
         "ruflin/elastica": "^5.3.5|^6.1.1"
     },
     "require-dev": {
         "doctrine/orm": "^2.5",
-        "doctrine/doctrine-bundle": "^1.6",
+        "doctrine/doctrine-bundle": "^1.6|^2",
         "doctrine/persistence": "^1.3.3",
         "doctrine/phpcr-bundle": "^1.3|^2.0",
-        "doctrine/phpcr-odm": "^1.4",
+        "doctrine/phpcr-odm": "^1.5",
```

Comment:
any reason to no longer allow 1.4 ?

Reply:
I don't remember, I'll try to revert it to ^1.4 and see if everything works